### PR TITLE
[skip changelog] chore: config: yet more lp2p removal from miner

### PR DESCRIPTION
**Pull request by ribasushi**
## Related Issues
https://github.com/filecoin-project/lotus/pull/12100

## Proposed Changes
There is no lp2p/pubsub in lotus-miner these days. Amazing how more and more code gets found...

### CODE REVIEW

File: node/config/def.go [hunk 1]
```diff
@@ -32,6 +32,18 @@ func defCommon() Common {
 		Backup: Backup{
 			DisableMetadataLog: true,
 		},
+	}
+}
+
+func DefaultDefaultMaxFee() types.FIL {
+	return types.MustParseFIL("0.07")
```
Comment (rjan90) [inline node/config/def.go:39]: Are we sure that this is only touching markets related? From my naive view it seems more "global" than that.

**Comment by rjan90** [comment]
> @rjan90 this one is a potential backport: it removes config/documentation that no longer does anything (part of the long-slog markets removal)

I put a backport-label on it for now. And also added a comment/question on one part which seemed to touch a bit more then just markets

File: node/builder_miner.go [hunk 2]
```diff
@@ -53,11 +53,7 @@ func ConfigStorageMiner(c interface{}) Option {
 	return Options(
 
 		Override(new(v1api.FullNode), modules.MakeUuidWrapper),
-		// Needed to instantiate pubsub used by index provider via ConfigCommon
-		Override(new(dtypes.DrandSchedule), modules.BuiltinDrandConfig),
-		Override(new(dtypes.BootstrapPeers), modules.BuiltinBootstrap),
-		Override(new(dtypes.DrandBootstrap), modules.DrandBootstrap),
-		ConfigCommon(&cfg.Common, build.NodeUserVersion(), false),
```
Comment (aarshkshah1992) [inline node/builder_miner.go:60]: @ribasushi Why are we getting rid of these 3 in the Miner ?

Override(new(dtypes.DrandSchedule), modules.BuiltinDrandConfig),
		Override(new(dtypes.BootstrapPeers), modules.BuiltinBootstrap),
		Override(new(dtypes.DrandBootstrap), modules.DrandBootstrap),
		
		Dosen't a Miner need Drand ?

**Review by aarshkshah1992 — APPROVED** [review]
lgtm

**Comment by aarshkshah1992** [comment]
@ribasushi Merging.

**Pull request merged**